Lib2025/src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -95,12 +96,12 @@ public double getBottomVelocity() {
         return m_bottomShooterMotor.getEncoder().getVelocity();
     }
 
-    public boolean isAtRPM(double RPM) {
-        return Utility.isWithinTolerance(getTopVelocity(), RPM, SHOOTER.RPM_TOLERANCE)
-                && Utility.isWithinTolerance(getBottomVelocity(), RPM, SHOOTER.RPM_TOLERANCE);
+    public boolean isAtVelocity(AngularVelocity vel) {
```

Comment:
You have two set velocity methods, but only one isAtVelocity(). It is unclear if this is comparing the velocity of the motor or the velocity of the wheel.

Reply:
Ended up removing the isAtVelocity method since it wasn't used anywhere except isAtTargetSpeed in the 2024 code